core/commands/root.go

```diff
@@ -72,6 +72,7 @@ TOOL COMMANDS
   update        Download and apply go-ipfs updates
   commands      List all available commands
   cid           Convert and discover properties of CIDs
+  log			Manage and show logs of running daemon
```

Comment:
Indentation doesn't match

Reply:
Damned tabulators, not sure why I used them 😅